projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/credentials/KerberosCredentialsRepository.java

```diff
@@ -57,7 +57,7 @@ private void checkDependencies() {
       ProcessExecutor executor = new ProcessExecutor();
       executor.readOutput(true);
       ProcessResult result = executor.command("/bin/sh", "-c", "which kadmin").execute();
-      if (result.getExitValue() != 0) {
+      if (result != null && result.getExitValue() != 0) {
```

Comment:
If result is null it would exit successfully. Are you sure that is what we want?

Reply:
good catch!